### PR TITLE
Added AI Assisted AutoCaptcha Feature

### PR DESCRIPTION
**Pull request by shailesh**
will close #252

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -244,19 +245,10 @@ def collect_user_details(request_header):
     )
     auto_book = "yes-please"
 
-
     print("\n================================= Captcha Automation =================================\n")
-    print("======== Caution: This will require a paid API key from anti-captcha.com =============")
 
     captcha_automation = input("Do you want to automate captcha autofill? (y/n) Default n: ")
```
Comment (shailesh) [inline src/utils.py]: default is changed to "Y" now

**Review by shailesh — COMMENTED** [review]
done

**Review by Nakul93 — APPROVED** [review]
All Good

**Comment by shailesh** [comment]
I'm super amazed by this change, tested extensively where Apollo opened just 20 slots in Bangalore for 18+

**Comment by tanujabameta** [comment]
This is very slow. Got late 3 times.

**Comment by Nakul93** [comment]
@tanujabameta by setting your auto-captcha preference to yes

**Comment by bombardier-gif** [comment]
@shailesh as you said you tested it for multiple slot, I am going ahead and merging this
Great job my man! 💯

**Comment by shailesh** [comment]
I also got vaccinated today <3, thank you, everyone. Now few issues to close for the second dose.

**Comment by bombardier-gif** [comment]
image
image

Thanks a ton for making this so crazy good. Stay safe!

**Comment by shailesh** [comment]
This is super fast

**Comment by madhu2504** [comment]
do i need to choose autofill captcha  option to make this work or chhose manually enter captcha option?

**Comment by shailesh** [comment]
choose auto, manul will open the popup

@bombardier-gif please help update the readme

**Comment by BumpyKnuckls** [comment]
@shailesh was about to tag you for the same. Please feel free to open a PR updating the README about the captcha, possibly in the `Before you start` section. Else I'll be happy to open one :)

**Comment by shailesh** [comment]
cool, I'll do that.

**Comment by BumpyKnuckls** [comment]
Thanks a lot :)

**Comment by sanket-gupta** [comment]
Hey
i doubt that he autocaptcha will keep working for long. If its so simple to break it, they will eventually change it.
We should keep the anticaptcha and 2captcha code as backup options

**Comment by shailesh** [comment]
Thanks will add them too
________________________________

**Comment by shailesh** [comment]
But in SVG case it is 💯 Percent going to work
________________________________

**Comment by shailesh** [comment]
Hey @sanket-gupta  As far as it is SVG it will work, the same logic is applied on their backend as well. Verified this after so many efforts. If it changes to something else then SVG everything will break including this.

**Comment by sanket-gupta** [comment]
but this SVG captcha solution is brilliant.
It has just increased the chance of booking manifolds!

**Comment by bombardier-gif** [comment]
There are a bunch of bugs reported. Please take a look to see if you PR caused it

**Comment by shailesh** [comment]
@bombardier-gif as far I could see there bugs came from 
1. Resolving conflicts for smarter way to select centre forgot the name of thr person some Rahul 
2. From Nakul's PR I guess. 

I had tested enough to make it bug free. And no bugs related to Captcha. And for OTP thing I guess it was non working day yesterday and lot of centre opening slots for weekend so my guess is that Cowin server was blocking the multiple requests or probably server going down.

**Comment by tanujabameta** [comment]
I got my vaccine today and I used Shailesh's branch to do it so I can
confirm it is working for multiple benificiaries and captcha is being
solved in less than a sec. Thanks a lot Shailesh.

**Comment by warlock2k** [comment]
This PR is just insane, thank you @shailesh

**Pull request merged**